app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -33,4 +49,19 @@ class SearchAndAddOrder extends React.Component {
   }
 }
 
-export default SearchAndAddOrder;
+const mapStateToProps = state => ({
+  inpatientCareSetting: state.careSettingReducer.inpatientCareSetting,
+  outpatientCareSetting: state.careSettingReducer.outpatientCareSetting,
```

Comment:
You may consider destructuring the state since careSetting object is the only thing you need from it.

Reply:
Makes sense to have the careSetting object destructured further. 
Done that now, thanks for the feedback.